sympy/core/tests/test_numbers.py

```diff
@@ -320,6 +320,19 @@ def test_Rational_new():
     assert Rational('.76').limit_denominator(4) == n3_4
     assert Rational(19, 25).limit_denominator(4) == n3_4
     assert Rational('19/25').limit_denominator(4) == n3_4
+    n2_3 = Rational(2, 3)
+    assert Rational('2/3') == n2_3
+    assert -Rational('-2/3') == n2_3
+    assert Rational('.68').limit_num_denom(3, 5) == n2_3
+    assert Rational(-17, 25).limit_num_denom(3, 5) == -n2_3
+    assert Rational('17/25').limit_num_denom(5) == Rational('17/25').limit_denominator(5)
+    assert Rational(7, 25).limit_num_denom(2, 3) == Rational(7, 25).limit_denominator(3)
+    assert Rational(17, 25).limit_num_denom(3, 1) == 1
+    assert Rational(17, 25).limit_num_denom(1, 5) == 1
+    assert Rational(25, 17).limit_num_denom(1, 5) == 1
+    assert Rational(25, 17).limit_num_denom(3, 5) == Rational(1, n2_3)
```

Comment:
I think this relies on additional evaluations and in this case there is no speed improvement compared to `Rational(3, 2)`.